invenio_opendata/testsuite/data/cms/cms-condition-data-Run2011A.xml

```diff
@@ -17,7 +17,12 @@
     <datafield tag="520" ind1=" " ind2=" ">
       <subfield code="a">Condition database includes non-event-related
       information (Alignment, Calibration, Temperature, etc) and parameters for
-      simulation/reconstruction/analysis software.</subfield>
+      simulation/reconstruction/analysis software.
+      <p> These files are needed when running a CMSSW job on the Open Data VM. They are automatically read from 
+      the /cvmfs/cms-opendata-conddb.cern.ch runtime, you do not need to download these files from here separately.
+      START53_LVA1 is to be used with the simulated data, and FT_53_LV5_AN1_RUNA is to be used with the collision data
+      of RunA 2011 (with the event range of the open data, the file for the full event range for 2011 are also stored 
+      under FT_53_LV5_AN1 on /cvmfs/cms-opendata-conddb.cern.ch)</p></subfield>
     </datafield>
```

Comment:
In order to use the HTML syntax in field values, you should use `CDATA`, for example:

```xml
<subfield code="a"><![CDATA[ <p> Here we go ... </p> ]]></subfield>
```